app/controllers/items_controller.rb

```diff
@@ -36,13 +36,23 @@ def edit
     end
 
     redirect_to user_items_path(current_user)
-
   end
 
   def destroy
+    @item = Item.find(params[:id])
```

Comment:
Think about where the `params[:id]` value is taken from - the URL.

Line 42 makes it possible for a logged-in attacker to tamper with the URL to delete other users `Item`s. 

An attacker could write a script to iterate over likely item ids and make `DELETE` HTTP requests to URLs containing those guessed ids.

One way to protect against this kind of attack is to query only for sensitive data that belongs to the logged-in user.

To do this, line 42 would make use of a `current_user`-scoped query:

```ruby
  @item = current_user.items.find(params[:id])
```
The above line will raise an `ActiveRecord::NotFound` error if an item cannot be found with the given id **and** belonging to `current_user`.